packages/ember/src/components/card.gts

```diff
@@ -26,7 +27,7 @@ export default class Card extends Component<CardSignature> {
 
   <template>
     <div
-      class="card p-4 shadow-sm"
+      class="card p-4 shadow-sm {{if @noBorder 'border-0'}}"
```

Comment:
Should we rename this to `hasBorder`? Default it to yes, but if `hasBorder={{false}}` then we add `border-0`?